pyte/streams.py

```diff
@@ -131,15 +138,17 @@ class Stream:
         "[^" + "".join(map(re.escape, _special)) + "]+")
     del _special
 
-    def __init__(self, screen=None, strict=True):
-        self.listener = None
+    def __init__(self, screen: Union[Screen | None] = None, strict: bool = True) -> None:
```

Comment:
| is only available since Python 3.10. Please revert.

Reply:
Whoops that's a typo, will fix

Reply:
Fixed